lldb/source/Breakpoint/BreakpointResolver.cpp

```diff
@@ -204,8 +205,18 @@ void BreakpointResolver::SetSCMatchesByLine(
     SearchFilter &filter, SymbolContextList &sc_list, bool skip_prologue,
     llvm::StringRef log_ident, uint32_t line, std::optional<uint16_t> column) {
   llvm::SmallVector<SymbolContext, 16> all_scs;
-  for (uint32_t i = 0; i < sc_list.GetSize(); ++i)
+  const bool ShouldQueryLanguageFilter =
+      GetBreakpoint()
+          ->GetTarget()
+          .GetIgnoreBreakpointsFromLanguageArtificialLocations();
+
+  for (uint32_t i = 0; i < sc_list.GetSize(); ++i) {
```

Comment:
can this be range-based?

Reply:
It can, I just didn't want to add to the diff. As much as github is saying this is new code, it's not new code...